### PR TITLE
PEP8 fixes so that make test-static can use make lint

### PR DESCRIPTION
**Pull request by netsettler**
* Renamed (disabled) some files in preparation for their later removal:
  * `.ebextensions` to `.ebextensions.DISABLED`
  * Various files in `deploy/` that seemed to have no callers:
    * `deploy/last_git_commit.py` to `deploy/last_git_commit.py.DISABLED`
    * `deploy/set_beanstalk_config.py` to `deploy/set_beanstalk_config.py.DISABLED`
    * `deploy/travis_after_all.py` to `deploy/travis_after_all.py.DISABLED`
* Make `make lint` actually invoke `flake8` instead of just saying it's coming soon.
* `pyproject.toml` uses `dcicutils 5.2.0` for new static check support, and `poetry.lock` is updated.
* Edits that were not just syntactic include:
  * Some unused variables in `test_search_ngram` in `src/encoded/tests/test_search.py` seemed to be tests of the wrong variable value, so I fixed a test (which fortunately was still passing).
  * Removed `build_xlsx_spreadsheet` in `batch_download_utils.py`. It does not appear to have any callers, and it has some problems that were not obvious how to fix. In particular there's an unused variable at the end, but I wonder if it doesn't want to return some value.
  * Commented out the content of `src/encoded/commands/extract_test_data.py`, which had numerous problems in the code (undefined functions, etc.) and couldn't possibly have worked. I don't think there's anything that uses this?
  * Rewrote some code in `src/encoded/commands/generate_items_from_owl.py` to call `dcicutils.command_utils.y_or_n` rather than using lower level primitives. I adjusted some prompts in the process.
  * In `src/encoded/commands/load_items.py`, rewrote some functions to require keyword-argument-calling because I don't think there are non-adjusted callers but I wanted to make sure that my addition of a `logger` argument to make some undefined varaibles work again was not going to cause a problem. I doubt anyone was calling this or they'd have complained about the undefined variables, so probably this is all fine. (Probably we should do auth stuff differently here, but I didn't bother with that.)
  * PEP8 doesn't like assigning lambda expressions to variables. I mostly do not think it's right about that, but the one case where we were doing it needed to be rewritten for other reasons, and I'd already done that rewrite in snovault, so I ported the fix from there.
  * Rewrote a few cases of `print` as `PRINT`. Maybe some as logger calls, too. Added static checkers for stray print statements. There are still a lot of them that need review. For now I just have it issuing a warning, not an error, while we work through those. I wrote ticket https://hms-dbmi.atlassian.net/browse/C4-929 on this.
* Removed a lot of unused imports, and alphabetized/merged many imports.
  * In some cases the unused imports were removed, and in others where they were "harder to find" names, I just commented them out while we let things shake out to make sure I didn't make an error.
  * In some cases I added an `ignorable` declaration for things where I expected a later change to bring back the need for the import.
  * In some cases I added `notice_pytest_fixtures` because the use of the name as a fixture is not lexically observable and PyCharm is bad about understanding what's going on.
* Rewrote some `'''...'''` doc strings as `"""..."""`.
* Reviewed unused variables.
  * Some were marked ignored.
  * Some were statements that could be removed entirely.
  * Some were side-effects where we could ignore return value and the left-hand side of the assignment could be removed.
* Adjusted whitespace in some expressions per PEP8.
* Removed some parentheses that PEP8 insisted were redundant.
  * Some of these were things like assert, which is not a function but was being "called" by doing `assert(...)`.
    * Same with `del(...)` that isn't a function either.
* Rewrote some `except:` as `except Exception:`.
* Updated some `.format()` calls to use f-strings.
* PEP8 doesn't like `l` as a variable name because it looks like a `1` in some fonts, so I rewrote some uses of that variable (usually as `lst` instead, though in a few cases there were obviously better names).
* Rewrote some `== True/False` as `is True/False` in testing.

### CODE REVIEW

**Comment by netsettler** [comment]
This passes all tests on my local machine:
```
=== 45 passed, 2 skipped, 1899 deselected in 119.57 seconds ===
=== 1551 passed, 16 skipped, 379 deselected in 637.03 seconds ===
=== 1 passed, 1945 deselected in 6.29 seconds ===
```

File: src/encoded/tests/test_search.py [hunk 1]
```diff
@@ -171,15 +169,15 @@ def test_search_ngram(workbook, es_testapp):
     assert len(res4['@graph']) == 3
     # search for full uuid
     res5 = es_testapp.get('/search/?type=Disorder&q=231111bc-8535-4448-903e-854af460b25').json
-    assert len(res4['@graph']) == 3
+    assert len(res5['@graph']) == 3
```
Comment (netsettler) [inline src/encoded/tests/test_search.py:172]: Note the unused variable `res5` was a symptom of testing the wrong value here.

File: src/encoded/tests/test_search.py [hunk 2]
```diff
@@ -171,15 +169,15 @@ def test_search_ngram(workbook, es_testapp):
...
     # uuid difference beyond 10
     res6 = es_testapp.get('/search/?type=Disorder&q=231111bc-89').json
-    assert len(res4['@graph']) == 3
+    assert len(res6['@graph']) == 3
```
Comment (netsettler) [inline src/encoded/tests/test_search.py:175]: Note the unused variable `res6` was a symptom of testing the wrong value here.

**Comment by netsettler** [comment]
Seeing these warnings on static tests that I mostly don't plan to deal with on this pass.
```
src/encoded/tests/test_misc.py::test_utils_debugging_artifacts_print
  /opt/hostedtoolcache/Python/3.7.14/x64/lib/python3.7/site-packages/dcicutils/qa_checkers.py:360: SyntaxWarning: 24 problems detected:
   In /home/runner/work/cgap-portal/cgap-portal/src/encoded/verifier.py, 3 calls to print.
   In /home/runner/work/cgap-portal/cgap-portal/src/encoded/visualization.py, 1 call to print.
   In /home/runner/work/cgap-portal/cgap-portal/src/encoded/authentication.py, 2 calls to print.
   In /home/runner/work/cgap-portal/cgap-portal/src/encoded/loadxl.py, 10 calls to print.
   In /home/runner/work/cgap-portal/cgap-portal/src/encoded/types/ingestion.py, 4 calls to print.
   In /home/runner/work/cgap-portal/cgap-portal/src/encoded/types/file.py, 2 calls to print.
   In /home/runner/work/cgap-portal/cgap-portal/src/encoded/types/family.py, 1 call to print.
   In /home/runner/work/cgap-portal/cgap-portal/src/encoded/types/workflow.py, 1 call to print.
    if_used=self.if_used)
```

**Review by willronchetti — APPROVED** [review]
These changes all seem harmless to me but should be deployed regardless just to double check

File: src/encoded/commands/extract_test_data.py [hunk 3]
```diff
@@ -1,153 +1,157 @@
...
-        traceback.print_exc()
-        # import pdb; pdb.post_mortem(tb)
-
-
-if __name__ == '__main__':
-    main()
+# NOTE: This file is full of calls to undefined functions, so I've commented it out for now.
+#       Is there someone who thinks there's a use for it and that we should try to repair it?
+#       -kmp 25-Sep-2022
+#
```
[the remark this answers is not in the document]
Comment (netsettler) [reply src/encoded/commands/extract_test_data.py:4]: @aschroed Do you know?

[the remark this answers is not in the document]
Comment (aschroed) [reply src/encoded/commands/extract_test_data.py:4]: I think this is leftover from encode.  It's still in their repo and hasn't been modified for a few years so I think it's probably safe to dispense with.

**Comment by netsettler** [comment]
Recent changes are merged again and tests passing still on my local machine.
```
=== 45 passed, 2 skipped, 1915 deselected in 126.52 seconds ===
=== 1550 passed, 16 skipped, 396 deselected in 584.61 seconds ===
=== 3 passed, 1959 deselected, 1 warnings in 5.78 seconds ===
```
The 1 warning is expected (mentioned earlier in this conversation), and is out of scope for this PR.

File: src/encoded/commands/extract_test_data.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (netsettler) [reply src/encoded/commands/extract_test_data.py:4]: Cool. Thanks.

**Comment by netsettler** [comment]
I've merged master, executed `poetry run order-schema-columns-and-facets`, and done some re-testing on my machine. Same results as before. Comparable times.
```
=== 45 passed, 2 skipped, 1915 deselected in 123.91 seconds ===
=== 1550 passed, 16 skipped, 396 deselected in 692.94 seconds ===
=== 3 passed, 1959 deselected, 1 warnings in 6.75 seconds ===
```

**Comment by netsettler** [comment]
I deployed this a couple days ago and it ran fine but I found I had not merged everything, and I also  had to do the reorderings, so the most recently mentioned tests include that. I re-ran the local testing twice on my machine and it was clean. I"m going to conclude that the GA problems are intermittent and are what Will's been working on.

**Pull request merged**